tasks/main.yml

```diff
@@ -11,11 +11,11 @@
   tags: [postgresql, postgresql-install]
 
 - import_tasks: install_yum.yml
-  when: ansible_pkg_mgr == "yum" and ansible_distribution == "RedHat" 
```

Comment:
the reason this was changed this way is that if you use os_family* for fedora it would come back as a red hat OS and run `install_yum.yml` besides the obvious failure of postgres 9.3 for fedora, travis build all of this successfully and worked for the vagrant files. I added pkg_mgr and distribution so it picks it up the correct one. if you leave pkg_mgr == "yum" it will run on fedora too.

Reply:
OK, so I've put those back in, and included CentOS too.

Reply:
@gclough maybe better something like !(I dont remember right syntax for logical `not`) `ansible_distribution == "Fedora"`?

Reply:
I guess we can either take either approach:

1) We only support RedHat and CentOS... so let's list them explicitly
2) We want to be generic, so let's use a negative test

The current code uses #1, but I'm happy enough to swap if that's the consensus.  Would someone else like to do that, so spread the credit for this work?

(also, I can't seem to squash my commits without screwing everything up.  I don't know why there are 43 commits in there!)